i18n/en.json

```diff
@@ -196,10 +202,11 @@
 	"namespaces-protection": "Which userright should be needed to edit this namespace?",
 	"namespaces-search": "Should this namespace be searchable by default?",
 	"namespaces-subpages": "Should this namespace have subpages enabled?",
+	"permissions-advanced": "Advanced settings",
```

Comment:
Hmm "Settings" on permissions seems a little confusing to me. But that's just me. Maybe it doesn't matter.